File: python translations/subroutine shiftmol.py
def shiftmol(c, n, c0, fac):
	"""
	Center the molecule at c0
	***
	Parameters:
	c: molecule coordinates (dimension 3*n)
	c0: given point in space 
	"""
	print("SHIFTMOL n = {} fac = {} c0= {} ".format(,n,fac,c0))
	for i in range(n):
		for k in range(3):
			cnew[k][i] = c[k][i] + fac*c0[k]
	return cnew

File: subroutines/all/subroutine shiftmol.f
      subroutine shiftmol(c,n,c0,cnew,fac)
      dimension c(3,n),cnew(3,n),c0(3)
C     Center the molecule at c0
c     print *,'SHIFTMOL n=',n,' fac=',fac,' c0=',c0
      do i=1,n
        do k=1,3
          cnew(k,i)=c(k,i)+fac*c0(k)
        end do
      end do
      return
      end
